src/python/pants/backend/python/goals/export_test.py

```diff
@@ -130,12 +131,14 @@ def test_export_venv_new_codepath(
             assert req_pex_dir == tmpdir
             assert req_pex_name == f"{resolve}.pex"
 
-            assert ppc0.argv[3:] == (
+            assert ppc0.argv[3:6] == (
                 "venv",
                 "--pip",
                 "--collisions-ok",
-                "{digest_root}",
             )
+            if py_resolve_format == PythonResolveExportFormat.mutable_virtualenv_with_non_hermetic_scripts:
+                assert ppc0.argv[6] == "--non-hermetic-scripts"
```

Comment:
```suggestion
            if py_resolve_format == PythonResolveExportFormat.mutable_virtualenv_with_non_hermetic_scripts:
                assert ppc0.argv[6] == "--non-hermetic-scripts"
            else:
                assert "--non-hermetic-scripts" not in ppc0.argv
```

Reply:
oh. good call. I'll make that adjustment.